drupal/sites/all/features/stats_charts/stats_charts.panels_default.inc

```diff
@@ -38,7 +38,7 @@ function stats_charts_default_panels_mini() {
     $pane->access = array();
     $pane->configuration = array(
       'admin_title' => 'Project stats overview',
-      'title' => 'Einstein@Home Progress',
+      'title' => variable_get('site_name', '') . " " . bts("Progress", array(), NULL, 'boinc:frontpage'),
```

Comment:
This wouldn't allow for translations where the order of the words need to change because it is translated as "Progress Einstein@Home". Better use a substitution instead of the concatenation so the translator can change the order of the two words.